content/300-accelerate/250-connection-pooling.mdx

```diff
@@ -4,39 +4,45 @@ metaTitle: 'Accelerate: Connection Pooling'
 metaDescription: "Learn about everything you need to know to use Accelerate's connection pooling."
 ---
 
-A [connection pool](https://en.wikipedia.org/wiki/Connection_pool#:~:text=In%20software%20engineering%2C%20a%20connection,executing%20commands%20on%20a%20database.) is a storage of database connections that can be reused for future requests to the database. When a new connection is requested, it is retrieved from the pool if one is available. Once the connection is no longer needed, it is returned to the pool for reuse.
+:::info
 
-[Connection pooling](https://www.prisma.io/dataguide/database-tools/connection-pooling) is important as it allows you to reuse existing connections instead of creating new ones, which can be an expensive operation. 
+To learn more about connection pools and connection pooling, be sure to check out [Prisma's Data Guide](https://www.prisma.io/dataguide/database-tools/connection-pooling) for a comprehensive overview on what connection pooling is, what it seeks to address, and how it works.
 
-The efficient management of database connections allows the database to process more queries without exhausting the available database connections, making your application more scalable. 
+:::
 
-Accelerate provides built-in connection pooling by default. By simply using Accelerate, you get the benefits of connection pooling without having to configure anything. However, you can also configure the connection pool to suit your needs.
+:::note
 
-<Admonition>
+For more information about the internal connection pool of Prisma ORM specifically, see our [ORM connection pooling documentation](/orm/prisma-client/setup-and-configuration/databases-connections/connection-pool).
 
-For more information about connection pooling in Prisma, see the documentation [here](/orm/prisma-client/setup-and-configuration/databases-connections/connection-pool).
+:::
 
-</Admonition>
+Accelerate provides built-in connection pooling by default. By using Accelerate, you get the benefits of connection pooling without having to configure anything. The efficient management of database connections allows the database to process more queries without exhausting the available database connections, making your application more scalable.
+
+In some cases, however, it may be beneficial to further configure connection pooling in order to optimize the performance of your application.
 
-### Default connection pool size
+### Connection Pooling in Accelerate
 
-Accelerate has a default `connection_limit` set to `10`.
+By default, Accelerate has a default connection limit of `10`. In most cases, this connection limit is sufficient. However, in high traffic or compute environments, you may need to [increase this value](#configuring-the-connection-pool-size). For the [Pro plan]() there is a maximum connection limit of `100`. For [Business plan]() users the maximum is `1000`.
```

Comment:
I think I would rather see this as a bulleted list than a paragraph. It would make it easier to see the differences between each plan. Let me know if that idea doesn't make sense.

Reply:
bulleted list or a table? What do you think?

Reply:
If you can make a table look nice then yea lets do that. I know we have had issues with tables in the past though.